scheduler/scheduler.go

```diff
@@ -283,6 +283,10 @@ func (s *eremeticScheduler) Error(_ sched.SchedulerDriver, err string) {
 }
 
 func nextID(s *eremeticScheduler) int {
+	if s.database.Count() > s.tasksCreated {
```

Comment:
There should really only be one source of truth for this value and if we're going to ask the database anyway I think we should always use what it gives us + 1